src/commands/tokens.js

```diff
@@ -0,0 +1,117 @@
+import colors from 'colors/safe.js';
+import { Logger } from '../logger.js';
+import { createApiToken, deleteApiToken, listApiTokens } from '../clever-client/auth-bridge.js';
+import { sendToAuthBridge } from '../models/send-to-api.js';
+import { getCurrent as getCurrentUser } from '../models/user.js';
+import { conf } from '../models/configuration.js';
+import dedent from 'dedent';
+import { promptPassword } from '../prompt-password.js';
+
+/**
+ * Create a new API token
+ * @param {Object} params - Function parameters
+ * @param {[string]} params.args - Command line args
+ * @param {Object} params.options - Command line options
+ * @param {'json'|'human'} params.options.format - Output format
+ */
+export async function create (params) {
+  const [apiTokenName] = params.args;
+  const { format } = params.options;
+
+  const user = await getCurrentUser();
+
+  const password = await promptPassword('Enter your password:');
+
+  let mfaCode;
+  if (user.preferredMFA === 'TOTP') {
+    mfaCode = await promptPassword('Enter your 2FA code:');
+  }
+
+  // Expire in 1 year
+  const dateObject = new Date();
+  dateObject.setFullYear(dateObject.getFullYear() + 1);
+  const expirationDate = dateObject.toISOString();
+
+  const tokenData = {
+    email: user.email,
+    password,
+    mfaCode,
+    name: apiTokenName,
+    expirationDate,
+  };
+  const createdToken = await createApiToken(tokenData).then(sendToAuthBridge);
+
+  switch (format) {
+    case 'json':
+      Logger.printJson(createdToken);
+      break;
+    case 'human':
+    default:
+      Logger.println(dedent`
+        ${colors.green('✔')} API token successfully created!
```

Comment:
a word about the fact that the token itself won't be showed anymore after that ?